File: python/bee1038.py
menu = {1: 4.00, 2: 4.50, 3: 5.00, 4: 2.00, 5: 1.50}

cod, qnt = map(int, input().split())

total = menu[cod] * qnt
print(f'Total: R$ {total:.2f}')

File: javascript/bee1038.js
var input = require("fs").readFileSync("stdin", "utf8");

const menu = {
	1: 4.0,
	2: 4.5,
	3: 5.0,
	4: 2.0,
	5: 1.5,
};

const code = input.split(" ")[0];
const n = input.split(" ")[1];
const total = menu[code] * n;

console.log("Total: R$ " + total.toFixed(2));
